www/src/pages/features/jamstack.js

```diff
@@ -38,58 +39,60 @@ const JamstackFeaturesPage = ({ data, location }) => {
   )
 
   return (
-    <Layout location={location} enableScrollSync={true}>
-      <Container>
-        <main id={`reach-skip-nav`}>
-          <Breadcrumb location={location} />
-          <FeaturesHeader />
-          <h3>Comparison</h3>
-          <p>
-            To see a filtered view of Gatsby with specific JAMstack
-            technologies, choose the technologies to compare and then press
-            Compare:
-          </p>
+    <Layout location={location}>
+      <PageWithSidebar location={location} enableScrollSync={true}>
```

Comment:
I don't know why `enableScrollSync` is set to true. This is the property that updates the sidebar active item when you scroll through the page, and is only used in the tutorial.

Reply:
Let's remove that - it feels like copy & paste case and sidebar in features don't have headings items just pages